### ISSUE TITLE
Arpabet sequence no noise

### ISSUE THREAD

**Issue by aaronlex** [NVIDIA/tacotron2#235]
I am trying to put in an ARPAbet sequence as text input for my trained model like:

"Turn left on {HH AW1 S S T AH0 N} Street"

but every time I try to use curly braces with ARPAbet the output is mostly silent. I have also tried separating the ARPAbet sequence into their own separate curly brackets (i.e. {HH} {AW1} {S}) but this doesn't work either. For the model to learn ARPAbet pronunciations properly should there be some ARPAbet sequences in the training text?

**Comment by rafaelvalle** [comment]
Did you train your model with ARPABET?

**Comment by aaronlex** [comment]
I see. Have you experimented with how much arpabet should be in training set so that the model learns it properly?

**Comment by rafaelvalle** [comment]
That's an hyperparameter that you can search based on validation loss and pronunciation, for example.

**Comment by sciai-ai** [comment]
> That's a hyperparameter that you can search based on validation loss and pronunciation, for example.

@rafaelvalle Which hyperparameter controls the arpabet?

Correct me I am wrong, should not the training text be automatically parsed based on the cmu dict and the valid symbols defined in the cmudict.py? Or are you suggesting that if one wants to synthesise sentences based on arpabet then the training texts should include arpabet symbols in curly braces as a preprocessing step "Turn left on {HH AW1 S S T AH0 N} Street"
